### DISCUSSION TITLE
deactivate the Vibration

### DISCUSSION THREAD

**Discussion by Jons-1** [skyline-emu/skyline#1968] [Q&A] [answered]
I would like that they added the function of "deactivate the Vibration",

some games that do not bring that option, and it makes it quite annoying (Like in captain toad, treasure)

**Comment by PixelyIon** [comment, marked as answer]
This functionality already exists; go to `Configure Controller #1` in Settings and reset the `Rumble Device`.

**Reply by Jons-1** [reply]
Thanks

**Reply by PekoTao** [reply]
Sorry, Pixelylon, I've 1 question... Can you give file from CI #1280?
I want to play Omori, Hollow knight and Inside in skyline, but my GPU is Mali, and i can't play... Build, what needs me, was expired, and now I cant download it... So, please, can you help me?...

**Reply by PekoTao** [reply]
No?...
